src/vuln_analysis/utils/standard_library_cache.py

```diff
@@ -92,6 +95,32 @@ def is_standard_library(self, package_name: str, ecosystem: Ecosystem | None) ->
                         package_name, self._hits, self._misses, eco_key)
             return found
 
+    def load_from_cache(self):
+        try:
+            path = self.cache_directory / self.cache_file_name     
+            if path.is_file():
+                with open(path, 'rb') as file:
+                    cache_ecosystem_to_set = pickle.load(file)
+                # Only replace if load was successful
+                self._ecosystem_to_set = cache_ecosystem_to_set
+                logger.info("Loaded runtime standard library cache from %s", path)
+        except Exception as e:
+            logger.error("Failed to load runtime standard library cache from %s: %s", path, e)
```

Comment:
@RedTanny First time it will always fail to load because the file will be empty, so there is a need for an additional informative log message in that case, because it's not an error.
```suggestion
        except EOFError as e:
            logger.info("runtime standard library cache still doesn't exist at path %s, skip loading", path, e)
            
        except Exception as e:
            logger.error("Failed to load runtime standard library cache from %s: %s", path, e)            
```